rfcs/Create-object-service.md

```diff
@@ -0,0 +1,53 @@
+# RFC  - Create an object service
+* Updates: [RFC 120](https://github.com/taskcluster/taskcluster-rfcs/blob/master/rfcs/0120-artifact-service.md)
+* Obsoletes: None
+* Comments: [#116](https://github.com/taskcluster/taskcluster-rfcs/pull/116), [#120](https://github.com/taskcluster/taskcluster-rfcs/pull/120)
+* Initially Proposed by: @jhford
+
+# Summary
+Taskcluster tasks often produce artifacts during the course of being run.
+Examples include browser distribution archives, json metadata and log files.
+We currently store artifacts in a single region of S3 and split management of
+artifacts between the Queue and allegedly Cloud Mirror.  
+
+This project is to design an object management api which pulls that functionality 
+out of the Queue and enables support of multiple clouds.
+
+This RFC is to decide on what the objects service is and its interface, not
+implementation details.
+
+# Motivation
+
+* We want to pay less, so we want to be able to store things in different regions and clouds
```

Comment:
I think we'll probably continue storing things in S3 if for no other reason than inertia, but being able to manage ingress from the other providers, and temporarily cache artifacts in a given provider where they might be needed repeatedly (e.g. builds for testing) are both standout use cases.

You should consider adding the most important use cases that you're using to justify this new service.